lib/new_relic/agent/instrumentation/active_record_helper.rb

```diff
@@ -229,11 +232,16 @@ module InstanceIdentification
           SLASH = "/".freeze
           LOCALHOST = "localhost".freeze
 
+          def adapter_from_config(config)
+            bare_name = NewRelic::Agent::Instrumentation::ActiveRecordHelper.bare_adapter_name(config[:adapter])
+            PRODUCT_SYMBOLS[bare_name]
+          end
+
           def host(config)
             return UNKNOWN unless config
 
             configured_value = config[:host]
-            adapter = PRODUCT_SYMBOLS[config[:adapter]]
+            adapter = adapter_from_config(config)
```

Comment:
Move the repeated use of this pattern out into a method, which the bare adapter name logic applied.